_includes/adops/adops-gam-setup.html

```diff
@@ -104,6 +107,24 @@ <h2 id="step-2-add-a-creative">Step 2. Add a Creative</h2>
 &lt;/script&gt;
 </code></pre></div></div>
 
+<p>Code provided above will work for all formats:</p>
+<ul>
+  <li>amp</li>
+  <li>banner</li>
+  <li>mobile</li>
+  <li>native</li>
+  <li>video (outstream video)</li>
+</ul>
+<p>Based on provided targeting <b>hb_format</b>, GAM will load different .js library for different ad format.  
+If you want to run only banner ads than you can put <b>banner.js</b> in the first line of your gam creative, like:</p>
+
+<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>&lt;script src = "https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/banner.js"&gt;&lt;/script&gt;
+  &lt;script&gt;
+  &lt;/script&gt;
+</code></pre></div></div>
+
+<p>But, if you plan to run multiple different formats on your page, then you should to create GAM creatives like described above with script path ending with <b>%%PATTERN:hb_format%%</b></p>
```

Comment:
Can you change this to: 

If you intend to run multiple ad formats on your page you will need to create the GAM creatives as described above with a script path ending with <b>%%PATTERN:hb_format%%</b>

Reply:
should be fixed